### ISSUE TITLE
Ubuntu: Trouble installing virtualenv; can't find skbuild.

### ISSUE THREAD

**Issue by stuaxo** [google/pytype#501]
I tried installing with poetry, and also had the same issue with pip.

Would it be possible to provide a manylinux .whl on pypi - this would make installation easier.

```
poetry add --dev pytype                                    stu@computer  6370  13:25:51 
Using version ^2020.1.24 for pytype

Updating dependencies
Resolving dependencies... (1.7s)

Writing lock file


Package operations: 5 installs, 0 updates, 0 removals

  - Installing decorator (4.4.1)
  - Installing networkx (2.4)
  - Installing importlab (0.5.1)
  - Installing ninja (1.9.0.post1)

[EnvCommandError]
Command ['/home/stu/.virtualenvs/toybot-scenegraph/bin/pip', 'install', '--no-deps', 'ninja==1.9.0.post1'] errored with the following return code 1, and output: 
Collecting ninja==1.9.0.post1
  Using cached ninja-1.9.0.post1.tar.gz (25 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/stu/.virtualenvs/toybot-scenegraph/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-p_xdbiw9/ninja/setup.py'"'"'; __file__='"'"'/tmp/pip-install-p_xdbiw9/ninja/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-p_xdbiw9/ninja/pip-egg-info
         cwd: /tmp/pip-install-p_xdbiw9/ninja/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-p_xdbiw9/ninja/setup.py", line 7, in <module>
        from skbuild import setup
    ModuleNotFoundError: No module named 'skbuild'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```

```
pip install pytype                                         stu@computer  6372  13:27:33 
Processing /home/stu/.cache/pip/wheels/6c/be/07/2a82fa74fd71a8302e45a8914c50fe7ea3b4f6a7759902464f/pytype-2020.1.24-cp37-cp37m-linux_x86_64.whl
Requirement already satisfied: attrs in /home/stu/.virtualenvs/toybot-scenegraph/lib/python3.7/site-packages (from pytype) (19.3.0)
Collecting ninja
  Using cached ninja-1.9.0.post1.tar.gz (25 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/stu/.virtualenvs/toybot-scenegraph/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-vwnul9pq/ninja/setup.py'"'"'; __file__='"'"'/tmp/pip-install-vwnul9pq/ninja/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-vwnul9pq/ninja/pip-egg-info
         cwd: /tmp/pip-install-vwnul9pq/ninja/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-vwnul9pq/ninja/setup.py", line 7, in <module>
        from skbuild import setup
    ModuleNotFoundError: No module named 'skbuild'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by rchen152** [comment]
Could you check what version of pip you're using? There was a bug in a recent pip release that caused this error (https://github.com/google/pytype/issues/495), but it was fixed in pip 20.0.2.

**Comment by stuaxo** [comment]
Cheers, I was on 20.0.1 that got me up and running.
